econml/dml.py

```diff
@@ -938,6 +941,17 @@ def __init__(self,
                          n_splits=n_splits,
                          random_state=random_state)
 
+    def shap_values(self, X, *, feature_names=None, treatment_names=None, output_names=None):
+        if self.featurizer is not None:
+            F = self.featurizer.transform(X)
+        else:
+            F = X
+        feature_names = self.cate_feature_names(feature_names)
+
+        return super()._shap_values(self.model_cate, F, feature_names=feature_names,
```

Comment:
Why isn't featurization handled within _shap_values? I suspect once _shap_values moves into a utility function in shap.py, we should push the featurization inside _shap_values (e.g. pass featurizer as an extra argument or None)

Reply:
Is this addressed? It's ok to keep as is.